examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1613,6 +1630,69 @@ impl FrontendNode {
 
         Ok(())
     }
+
+    fn test_grpc_server_second(&mut self) -> TestResult {
+        // Create a second gRPC server Node on a different port.
+        let result = oak::grpc::server::init("[::]:8181");
+        expect_matches!(result, Ok(_));
+        let handle = result.unwrap();
+        // Close the only read-handle for the invocation handle, which should
+        // trigger the gRPC server pseudo-Node to terminate (but we can't
+        // check that here).
+        expect_eq!(Ok(()), oak::channel_close(handle.handle));
+        Ok(())
+    }
+
+    fn test_grpc_server_invalid_address(&mut self) -> TestResult {
+        // Attempt to create an additional gRPC server with an invalid local address.
+        expect_eq!(
+            Err(OakStatus::ErrInvalidArgs),
+            oak::grpc::server::init("10 Downing Street")
+        );
+        Ok(())
+    }
+
+    // Under the covers, the oak::grpc::server::init() helper performs
+    // several steps. Test various failure conditions for each of those steps.
+    // We can't really check any failures, but hopefully nothing crashes...
```

Comment:
Is it possible to read or write from the passed handle and to check if it fails due to non-existing Node?